### PR TITLE
Replace pre-calculated embedding with auto-embedding 

### PR DESCRIPTION
**Pull request by JoelPasvolsky**
Takes advantage of Advantage's shorter chains and Ocean's improvements to chain strength defaults to drop pre-calculated embeddings

### CODE REVIEW

**Review by arcondello — APPROVED** [review]

File: 01-factoring-overview.ipynb [hunk 1]
```diff
@@ -411,14 +411,9 @@
...
-    "# To see helper functions, select Jupyter File Explorer View from the Online Learning page\n",
-    "from helpers.embedding import embeddings\n",
-    "from dwave.system import FixedEmbeddingComposite\n",
-    "\n",
-    "sampler_fixed_embedding = FixedEmbeddingComposite(sampler, embeddings[sampler.solver.name])\n",
+    "from dwave.system import EmbeddingComposite\n",
     "\n",
-    "print(\"Mapping from BQM variables to qubits for the {} system:\\n\\n\\t{}\".format(\n",
-    "      sampler.solver.name, sampler_fixed_embedding.embedding))"
+    "sampler_embedded = EmbeddingComposite(sampler)"
```
Comment (arcondello) [inline 01-factoring-overview.ipynb]: Maybe `embedding_sampler`, the sampler has not been "embedded". I know we do this in lots of places...

Comment (JoelPasvolsky) [reply 01-factoring-overview.ipynb]: Done

File: 01-factoring-overview.ipynb [hunk 2]
```diff
@@ -390,10 +388,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from dwave.system.samplers import DWaveSampler\n",
+    "from dwave.system import DWaveSampler\n",
     "\n",
     "# Use a D-Wave system as the sampler\n",
-    "sampler = DWaveSampler(solver={'qpu': True})  "
+    "sampler = DWaveSampler() \n",
```
Comment (randomir) [inline 01-factoring-overview.ipynb:394]: This will work as expected only in Ocean 3.0.0+

**Review by randomir — COMMENTED** [review]
Nice. Just lower-bound the SDK to 3.0 in requirements.

Unrelated, why the "01-" prefix for this lonely notebook?

**Comment by randomir** [comment]
> [..] Just lower-bound the SDK to 3.0 in requirements.

Oh, I see you already did that in #10. Just waiting for @arcondello's review.

File: 01-factoring-overview.ipynb (hunk 2 above)
Comment (JoelPasvolsky) [reply 01-factoring-overview.ipynb:394]: Yes, and we'd only want to drop pre-calculated embeddings for SDKs that have the improved chain-strength default behavior. This PR will come after https://github.com/dwave-examples/factoring-notebook/pull/10

Comment (randomir) [reply 01-factoring-overview.ipynb:394]: Merged now.

File: 01-factoring-overview.ipynb [hunk 3]
```diff
@@ -178,9 +178,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Use Ocean's [dwavebinarycsp](https://docs.ocean.dwavesys.com/en/stable/docs_binarycsp/sdk_index.html)  to convert the binary CSP to a BQM. \n",
-    "\n",
-    "The code cell below does so and prints out the BQM's coefficients, the inputs used to program the D-Wave system. As noted, more than one BQM can represent an AND gate, so the BQM generated here does not have to match the one written above."
+    "Use Ocean's [dwavebinarycsp](https://docs.ocean.dwavesys.com/en/stable/docs_binarycsp/sdk_index.html)  to convert the binary CSP to a BQM, then print out the BQM's coefficients, the inputs used to program the D-Wave system. As noted, more than one BQM can represent an AND gate, so the BQM generated here does not have to match the one written above."
```
Comment (mdecandia) [inline 01-factoring-overview.ipynb]: There's a double whitespace before "to convert" (it isn't new)

**Pull request merged**